save-preprocessor/src/main/kotlin/org/cqfn/save/preprocessor/controllers/DownloadProjectController.kt

```diff
@@ -441,7 +439,7 @@ class DownloadProjectController(private val configProperties: ConfigProperties,
         val saveProperties: SaveProperties = decodeFromPropertiesFile<SaveProperties>(propertiesFile)
             .mergeConfigWithPriorityToThis(defaultConfig())
         return propertiesFile.parentFile
-            .resolve(saveProperties.testRootPath!!)
+            .resolve(saveProperties.testFiles!!.firstOrNull() ?: ".")
```

Comment:
Why for example just `testFiles!!.first()` is not enough?

Reply:
Because `SaveProperties` don't have default value for `testFiles`, and here we are using only properties file, while users would usually specify root dir via CLI. Therefore in this case we will very likely face an NPE.